sdk/include/psdk/winerror.h

```diff
@@ -1873,13 +2686,21 @@
 #define DNS_STATUS_FQDN                                    9557
 #define DNS_STATUS_DOTTED_NAME                             9558
 #define DNS_STATUS_SINGLE_PART_NAME                        9559
-#define DNS_ERROR_INVALID_NAME ERROR_INVALID_NAME
```

Comment:
In the official MS PSDK winerror.h, there are:
```
#define DNS_STATUS_PACKET_UNSECURE DNS_ERROR_UNSECURE_PACKET

#define DNS_ERROR_NO_MEMORY            ERROR_OUTOFMEMORY
#define DNS_ERROR_INVALID_NAME         ERROR_INVALID_NAME
#define DNS_ERROR_INVALID_DATA         ERROR_INVALID_DATA
```
So in particular, please restore this DNS_ERROR_INVALID_NAME.